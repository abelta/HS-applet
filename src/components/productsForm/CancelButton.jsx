import React from 'react';
import PropTypes from 'prop-types';

const CancelButton = ({ onClick }) => <button onClick={onClick}>Cancel</button>;

CancelButton.propTypes = {
  onClick: PropTypes.func.isRequired,
};

export default CancelButton;
