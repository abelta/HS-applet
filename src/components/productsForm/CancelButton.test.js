import React from 'react';
import renderer from 'react-test-renderer';
import CancelButton from './CancelButton';

describe('CancelButton', () => {
  const tree = renderer.create(<CancelButton onClick={jest.fn} />).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
