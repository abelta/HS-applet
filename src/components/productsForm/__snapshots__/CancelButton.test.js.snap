// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CancelButton renders correctly 1`] = `
<button
  className="btn-flat waves-effect waves-light red white-text"
  onClick={[Function]}
  style={
    Object {
      "borderRadius": 0,
    }
  }
>
  Cancel
</button>
`;
